core/src/test/java/com/epam/esm/service/impl/GiftServiceImplTest.java

```diff
@@ -0,0 +1,148 @@
+package com.epam.esm.service.impl;
+
+import com.epam.esm.dao.GiftDao;
```

Comment:
One comment related to all test classes for services:
please check the whole returned objects at least because your services uses convertors.
so you has to check not only the service methods but also convertor's methods as this methods are static and you can't mock them